src/Plugin.php

```diff
@@ -2,12 +2,15 @@
 
 declare(strict_types=1);
 
-namespace Pest\PluginName;
-
-/**
- * @internal
- */
-final class Plugin
-{
-    // ..
+use Livewire\Testing\TestableLivewire;
+use Pest\Livewire\InteractsWithLivewire;
+
+Pest\Plugin::uses(InteractsWithLivewire::class);
+
+if (!function_exists('livewire')) {
```

Comment:
I think this function don't makes sense as livewire already gives static access to test liveware components.